.eslintrc.js

```diff
@@ -73,7 +73,6 @@ module.exports = {
       'linebreak-style': [2, 'unix'],
       'new-parens': 2,
       'no-array-constructor': 2,
-      'no-bitwise': 2,
```

Comment:
I don't think removing bitwise as an exception makes that much sense or as a checkstyle. The intention of this checkstyle is to accidentally make typos when writing logical operations. Since we actually have bitwise now, we should just enable it everywhere. Config and other type of checkstyles, we can turn off